crates/bevy_ecs/src/world/error.rs

```diff
@@ -1,12 +1,62 @@
-//! Contains error types returned by bevy's schedule.
+//! Contains error types returned by methods on [`World`].
+//!
+//! [`World`]: crate::world::World
+
+use std::any::TypeId;
 
 use thiserror::Error;
 
-use crate::schedule::InternedScheduleLabel;
+use crate::{component::ComponentId, prelude::*, schedule::InternedScheduleLabel};
 
 /// The error type returned by [`World::try_run_schedule`] if the provided schedule does not exist.
 ///
 /// [`World::try_run_schedule`]: crate::world::World::try_run_schedule
 #[derive(Error, Debug)]
 #[error("The schedule with the label {0:?} was not found.")]
 pub struct TryRunScheduleError(pub InternedScheduleLabel);
+
+/// The error type returned by [`World::get_reflect`] and [`World::get_reflect_mut`].
+///
+/// [`World::try_run_schedule`]: crate::world::World::try_run_schedule
```

Comment:
This link doesn't appear to be used in these doc comments

Reply:
Good catch, left over from copy & paste :+1: